regulations/static/regulations/css/less/layout.less

```diff
@@ -4,6 +4,11 @@
  layout.less defines the structural layout of the site
  */
 
+// focus for accessibliity tabbing
+a:focus {
+    border: 1px dotted @black;
+}
```

Comment:
Related, note the missing bottom border and the one-pixel offset.
<img width="231" alt="screen shot 2016-06-07 at 1 44 22 pm" src="https://cloud.githubusercontent.com/assets/326918/15868476/fffaf996-2cb5-11e6-88b5-b151740c4190.png">

I'm guessing we should stick with outline and figure out why it isn't being applied consistently.